README.md

```diff
@@ -134,11 +134,14 @@ When read into a UVData object, the `history` string will contain information on
 Example `obsparam` configuration files may be found in the `reference_simulations` directory.
 
 1. Install from github or pip.
-2. Run off of a parameter file with 20 MPI ranks:
+2. Run off of a parameter file with 4 MPI ranks:
 ```
-mpirun -n 20 run_param_pyuvsim.py reference_simulations/obsparam_1.1.yaml
+mpirun -n 4 python scripts/run_param_pyuvsim.py reference_simulations/first_generation/obsparam_ref_1.1.yaml
 ```
 
+If you have not installed pyuvsim and want to run directly on the development code, you might need to point `PYTHONPATH` to the appropriate root pyuvsim directory to avoid Module not found errors.
```

Comment:
I believe this is already covered by the `Developer Installation` section. Running `pip install --no-deps -e .` is the recommended way to do this.

Reply:
Right. Removed this.